### DISCUSSION TITLE
Will it be possible to migrate from CentOS 7 to AlmaLinux ?

### DISCUSSION THREAD

**Discussion by ChristianTremblay** [AlmaLinux/almalinux-deploy#5] [Q&A] [answered]
If not, what is the suggested upgrade path ?

**Comment by ChristianTremblay** [comment, marked as answer]
Got the answer in the [Q&A](https://www.youtube.com/watch?utm_campaign=AlmaLinux+Beta+Announcement&utm_medium=email&_hsmi=111583758&_hsenc=p2ANqtz-9xlwbNBOa0XPq6zmXT4jjCVUC2rIk-vkAWYCdJyqCgDKS11SsY2f8oLg-MPxTJEaNnd9biTiBfD8YabnwoW99eceKM96GrEwCgetdnG1MHhpG9ePE&utm_content=111583758&utm_source=hs_email&v=ThavG6o7BrM&feature=youtu.be) of the Webinar

Must go to Centos 8 then migrate to AlmaLinux

**Comment by gregarios** [comment]
I would love a direct-upgrade from CentOS 7 too.

**Reply by taidos** [reply]
The only release available of AlmaLinux is for 8 version. 7 was not done and if it will be will be one pain to get new mirrors, existing rpm moving to the mirrors, will be months to prepare and i believe for one older operating system doesnt worth it. Better you upgrade your system allready rather later
